### ISSUE TITLE
App demo within the docs

### ISSUE THREAD

**Issue by SimonBiggs** [pymedphys/pymedphys#1687]
Heya @Matthew-Jennings,

If you would like, the app demo is now live again over at:

https://share.streamlit.io/pymedphys/pymedphys/main/app.py

That can be included in the docs if you would like?

**Comment by KeoJam** [comment]
But can not  run any demo except dicom pseudonymise

**Comment by SimonBiggs** [comment]
> But can not run any demo except dicom pseudonymise

Yup, each of these applications were built to support the clinic that the Medical Physicist was in. And certainly the "last mile" of making it easily demoable has not yet been undergone.

But, that is also why the "beta" and "alpha" labels were designated. Potentially there might be benefit of having a "toggle switch" which by default hides all applications that are less than beta quality?

Is this something you, @KeoJam, might be interested in setting up?

Essentially, all that would be required, is the setting up of a [streamlit checkbox](https://docs.streamlit.io/library/api-reference/widgets/st.checkbox), and then use it to only pull in "stable apps", and not "experimental apps" into the application options:

https://github.com/pymedphys/pymedphys/blob/17c9d93f4f9deef51003f0f86773ac506cd62693/lib/pymedphys/_streamlit/index.py#L138

**Comment by KeoJam** [comment]
yeah, I am interested in ICOM,but i try use pymedphys._delivery.Delivery.from_icom and got too many mus, gantrys, collimators, mlcs, jaws  that not match the plan. Is it only way to find the real gantry, collimator, mlc, jaw by compare  mu(one from plan's beam's control point  on from icom log files)?

**Comment by SimonBiggs** [comment]
> got too many mus, gantrys, collimators, mlcs, jaws that not match the plan

That's due to the iCOM stream being time resolved as opposed to control point resolved. I have found that a good way to compare the iCOM stream to the plan is via the mudensity tool.

Here's an example where mudensity is being used to compare trf and dicom:

https://github.com/pymedphys/pymedphys/issues/989#issuecomment-673198192

But, you can use the same approach to compare icom to dicom or other options also.

**Comment by SimonBiggs** [comment]
@lipteck also had a poster at the Singapore World Congress on this:

https://www.linkedin.com/feed/update/urn:li:share:6942750559179272195/

@lipteck, might you be in a position to give @KeoJam a hand?

**Comment by KeoJam** [comment]
Thanks, I'll try.

**Comment by SimonBiggs** [comment]
> Thanks, I'll try.

If you have any questions, jump onto the following mailing list and ask away:

https://pymedphys.discourse.group/
